Add additional support for parachutes

amy-chen-skydio opened the pull request:
- Add messages to report parachute status and information
 - Add commands to arm with additional trigger methods and to shutdown the module

amy-chen-skydio committed:
Add additional support for parachutes
 - Add messages to report parachute status and information
 - Add commands to arm with additional trigger methods and to shutdown the module

jenke-skydio reviewed (commented):
lgtm.  Would be good to get another reviewer before approved

abe-skydio reviewed (approved):
lgtm

amy-chen-skydio committed:
Merge branch 'mavlink:master' into skydio-add-parachute-support

amy-chen-skydio committed:
Update common.xml
 - Add `MAV_CMD_SET_PARACHUTE_ARM_ALTI` for setting the ATS arm altitude
 - Add `PARACHUTE_ERROR_FLAGS_GS_ERROR` and `PARACHUTE_ERROR_FLAGS_SUBSYSTEM_ERROR` in `PARACHUTE_ERROR_FLAGS`

amy-chen-skydio committed:
Update common.xml

amy-chen-skydio committed:
Merge branch 'mavlink:master' into skydio-add-parachute-support
